### PR TITLE
Homework for arrays and strings is done.

### PR DESCRIPTION
**Pull request by mstrakh**
Homework for arrays and strings is done.

### CODE REVIEW

File: tasks/arrays/arrays.js [hunk 1]
```diff
@@ -51,7 +61,11 @@ function isAnyFishPerson(chars) {
  * @return {number} - minimum element index
  */
 function minItem(arr) {
-    //PLACE YOUR CODE HERE
+    const arr1 = arr.map(el => el);
```
Comment (dshaplyko) [inline tasks/arrays/arrays.js:64]: Very interesting solution :)

File: tasks/strings/strings.js [hunk 2]
```diff
@@ -4,23 +4,40 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof str !== "string") {
+    return 'This is not a string!';
+  }
+  const stringArr = str.split("");
+  stringArr.reverse();
+  return stringArr.join("");
```
Comment (dshaplyko) [inline tasks/strings/strings.js:12]: can be done in one-line:
return str.split('').reverse().join('');

File: tasks/strings/strings.js [hunk 3]
```diff
@@ -4,23 +4,40 @@
...
  * 1. Calculate a century by given year
  * if a number is passed
-* @param {number} year
-*/
+ * @param {number} year
+ */
 function centuryFromYear(year) {
-
+  const yearAsInt = parseInt(year);
+  centuryPart = Math.floor(yearAsInt / 100);
```
Comment (dshaplyko) [inline tasks/strings/strings.js:22]: Nope, please use Math.ceil() and you won't need any additional actions

File: tasks/strings/strings.js (hunk 2 above)
Comment (mstrakh) [reply tasks/strings/strings.js:12]: Yep, really. :)

File: tasks/strings/strings.js (hunk 3 above)
Comment (mstrakh) [reply tasks/strings/strings.js:22]: Thanks, it would be much better.

**Pull request closed without merging**